### ISSUE TITLE
"the judge": scene-evoking?

### ISSUE THREAD

**Issue by nschneid** [UniversalConceptualCognitiveAnnotation/docs#25]
A student-authored example that we felt fairly confident about, except for one issue: p. 18 of the guidelines mentions that "driver" is a scene-evoking noun in "taxi driver". So should all uses of "judge" (the profession) be scene-evoking? Should it be P+A, because "judge" denotes both the scene and one of its participants? Should there be an implicit second participant, that which is judged?

![img_20180913_142052](https://user-images.githubusercontent.com/985263/45511898-3c405480-b76c-11e8-8c34-51704d629aad.jpg)

**Comment by dotdv** [comment]
We discussed this a few days ago with one of the annotators: we said the profession should be annotated just P (without A or IMP A). 
so for example: [[Judge_P]_E Harris_C]_A ruled in John's favor

**Comment by nschneid** [comment]
So what are the criteria for annotating P+A or S+A on a unit?

FrameNet has a concept of "denoted frame element"—basically if a noun both evokes a scene and denotes one of its roles. I figured that's what P/S+A was for.

**Comment by omriabnd** [comment]
Resolution: judge should be P+A, not if it's a title. Cite FrameNet in a footnote.

**Comment by nschneid** [comment]
What about "the handwritten letter to John"? Is "letter" scene-evoking because it implies sending a message to John?

**Comment by dotdv** [comment]
One of the annotators asked me:
[[[the_E judge_C]_P+A]_A listened to the witness.
or
[[the]_E [judge_P+A]_C]_A listened to the witness.

I saw the latter being used on several occasions also by annotators, 
but I think that the first option is more consistent with other instructions we give, namely that Cs can't be scenes and that determiners should be included inside the P/S  (articles and demonstratives only I think). 
So in a more complex example: 
[[The_E]_{P+A-} tired_D [judge_C]_{-P+A}]_A listened to the witness

This will make it similar to how we generally tend to treat determiners in noun scenes : 
We saw_P [[a_E]_{P-} great_D [show_C]_{-P}]_A 
This decision of still including the determiner within the P/S even when there's an adjective (D in this case) in between is something we verbally agreed upon with the Eng annotators, but haven't yet clarified in the guidelines.

**Comment by omriabnd** [comment]
I agree with you. The first option is better.

**Comment by dotdv** [comment]
There is a problem though. How should we deal with cases like:
  "[The tall judge]_A was able to reach for the book on  the top shelf"
Since 'tall' doesn't modify the process of judging, marking it [[the]_{P+A-} tall_D judge{-P+A}] seems problematic, no? or do you think it somehow acceptable?
If indeed we see it as a problem, then it seems that in such cases we will need to first start with C and Es,  but then is there a way to avoid C scenes?:
[The_E [tall_S (judge)_A]_E [judge_{P+A}]_C]_A was able to reach for the book on the top shelf

Maybe a possible alternative is to internally mark the Remote as P: "the_E [tall_S (judge_P)_A ]_E [judge]_C"
But this would only be optional if it were possible to internally annotate Remotes..

**Comment by nschneid** [comment]
Good point. Let's make it even more interesting and add a modifier that should go in the scene evoked by "judge".

Suppose we're talking about a dog show, and thus refer to a "dog judge" (somebody who judges the dogs in the competition).

- [[the_E [tall_S (judge)_A]_E judge_C]_P dog_A (judge)_A]_A was able to reach...
   Is this too complex of a P?

- [the_E [tall_S (judge)_A]_E [dog_A judge_P+A]_C]_A was able to reach...
   This feels like the more natural structure given the compositionality of the sentence. But it has a C-scene.

**Comment by omriabnd** [comment]
I think we should allow C Scenes, but instruct annotators to use them very
sparingly. For instance,

experienced_D taxi_A driver_P+A

and *not*:

experienced_E [taxi driver]_C

**Comment by nschneid** [comment]
OK, so

- the driver: [the_E driver_C]_P+A
- the taxi driver: taxi_A [the_E driver_C]_P+A
- the driver of the taxi: [the_E driver_C]_P+A [of_R the_E taxi_C]_A
- the experienced taxi driver: experienced_D taxi_A [the_E driver_C]_P+A (presumably "experienced" relates to the manner of driving)

But

- the tall taxi driver: the_E [tall_S (driver)_A]_E [taxi_A driver_P+A]_C
- the taxi driver who swims: the_E [taxi_A driver_P+A]_C [who_R [swims_P (driver)_A]_C]_E

?

**Comment by omriabnd** [comment]
Yes. Exactly.

**Comment by dotdv** [comment]
added the option of "C-scenes" under subsection "Model 3: Inter-Scene relations"
*mentioned that they should be used sparingly

**Comment by nschneid** [comment]
> the experienced taxi driver: experienced_D taxi_A [the_E driver_C]_P+A (presumably "experienced" relates to the manner of driving)

A consequence of this is we don't really know whether the participant-entity or the scene itself is the head of the unit (implicitly, a normal Scene unit is headed by a P/S). Thus "the driver scared her" and "the driving scared her" would be structurally similar:

- [the_F driver_P+A]_A scared her
- [the_F driving_P]_A scared her

Are we OK with that? I guess the assumption could be that in a P+A unit, the A is the category that applies to the full word: "driver" as a whole refers to the person. Presumably zero-derived role words like "boss" and "engineer" would have P+A for the noun (entity) use and just P for the verb (event) use.

What about more complicated derivational morphology where a noun for a phenomenon or quality is derived from a participant: "consumerism", "criminality", "demagoguery", "musicianship"? Should these be simply P/S, ignoring the fact that they are derived because the participants are generic?

**Comment by dotdv** [comment]
> added the option of "C-scenes" under subsection "Model 3: Inter-Scene relations"
> *mentioned that they should be used sparingly

So far we know that a role/profession should first be marked a C-scene if one of its siblings does not modify the process that the role evokes (tall judge or judge who swims). 

But is it also true to say that if a role/profession is the head of a RC we necessarily have to mark the role C?
Related streusle example: Davie_A is_S [[a_F [patient_C and_N methodical_C]_D teacher_P+A]_C [who has a great ear and sensitivity for his students passions]_E]_A?

And the next question would be what to do if the same scene appears as a top level scene? 
[patient_C and_N methodical_C]_D teacher_P+A [who has a great ear...]_?]_H?

**Comment by omriabnd** [comment]
I agree with both analyses you suggested.

**Comment by nschneid** [comment]
> Davie_A is_S [[a_F [patient_C and_N methodical_C]_D teacher_P+A]_C [who has a great ear and sensitivity for his students passions]_E]_A?

Looks OK to me, though I worry we could get cases like "tall and patient teacher" where only one adjective describes the act of teaching.

I don't understand how the second example is different.
